css/styles.css

```diff
@@ -748,8 +801,22 @@ a {
   transform: 0.8s;
 }
 
-.nav-link .navigation-bar {
-  display: none;
+#navbar a {
+  color: #fff;
+}
+
+.nav-overlay {
+  flex-direction: column;
+  height: 90%;
+  width: 100%;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background: #4756df;
+  overflow-y: hidden;
+  transition: 0.5s;
+  opacity: 0.96;

```

Comment:
- [ ] Your did a great job with the design of your mobile menu, it looks almost identical to the Figma template but on some screen it does not completely cover the entire screen. Kindly fix that.
Your design:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/61919617/181260472-50e2451e-43f4-4290-a61b-6e8fc0c14bdd.png">